io.github.johannesboehler2.BmiCalculator.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id" : "io.github.johannesboehler2.BmiCalculator",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "bmicalculator",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```

Comment:
this is not allowed and the build will not start with it.

Reply:
> this is not allowed and the build will not start with it.
Do you mean "append-path" and "--share=network"?

Reply:
No only the build-args part.